lib/common.sh

```diff
@@ -2,18 +2,19 @@
 
 [[ ":$PATH:" != *":/usr/local/go/bin:"* ]] && PATH="$PATH:/usr/local/go/bin"
 
+export USER="$(whoami)"
 # Verify that passwordless sudo is configured correctly
-if ! sudo -nl | grep -q '(ALL) NOPASSWD: ALL';then
-    echo "ERROR: metal3-dev-env requires passwordless sudo configuration!"
-    exit 1
+if [[ "$USER" != "root" ]]; then
+    if ! sudo -nl | grep -q '(ALL) NOPASSWD: ALL';then
+        echo "ERROR: metal3-dev-env requires passwordless sudo configuration!"
+        exit 1
+    fi
```

Comment:
Please consider line https://github.com/metal3-io/metal3-dev-env/blob/048058d20b8546816ac9d3f965c77077fcbc4d54/lib/common.sh#L289

Reply:
Not sure I just thought might include this restriction when checking if the user is root for the clarity of the script